tools/fixedpointarithmetic/src/main/java/dk/alexandra/fresco/fixedpoint/FixedNumeric.java

```diff
@@ -0,0 +1,135 @@
+package dk.alexandra.fresco.fixedpoint;
+
+import dk.alexandra.fresco.framework.DRes;
+import dk.alexandra.fresco.framework.builder.ComputationDirectory;
+import java.math.BigDecimal;
+
+/**
+ * Basic interface for fixed point numeric applications.
+ */
+public interface FixedNumeric<SFixedT extends SFixed> extends ComputationDirectory {
```

Comment:
Right now we are not entirely sure how the fixedpoints will actually be used. The operations implemented currently are the ones we are fairly sure will be needed for some simple machine learning. If we need conversion, it is easy to implement (just a division or multiplication). The same goes for an "advanced" computation directory. If/as we get more operations, it might become relevant, but currently I believe it makes sense with a single fixed numeric computation directory.

Reply:
My point was also that since it is easy to implement conversions, we might as well do it, but I guess we're going for the principle of: are we using it? If not, throw it away, so implementing something that might not be used is probably a bad idea now that I think twice.